src/esp/metadata/MetadataMediator.cpp

```diff
@@ -312,7 +321,7 @@ attributes::SceneAttributes::ptr MetadataMediator::getSceneAttributesByName(
         // 4.  Existing stage config/asset on disk, but not in current dataset.
         //    In this case, a stage attributes is loaded and registered, then
         //    added to current dataset, and then 3. is performed.
-        LOG(INFO)
+        !Cr::Utility::Debug{}
```

Comment:
About the `!` that prepends the message with file/line info, just a personal opinion -- I personally find it to be unnecessary noise, given that the error message contains the function name already. Consequently this is also a reason why there's `CORRADE_ASSERT()` with a message but no file/line info and `CORRADE_INTERNAL_ASSERT()` with file/line info but no message:

- user-facing messages should contain enough info to allow the user fix the root cause that caused the message and shouldn't need file/line info that's just noise to the user (if I get a bugreport in Magnum with certain error and it's not immediately clear from the message, it's enough to just `grep` for the message text, plus file/line info gets outdated *very soon* if users tend to prefer stable releases)
- on the other hand, file/line info is very useful for immediate "printf debugging" and developer-facing messages, where knowing the exact location where the error happened is most useful, not matter how beautifully crafted the human-readable error message is

Reply:
Thanks.  So perhaps I'll just use the ! messages for debug, and change the warning and error messages to not have the bang?

Reply:
I would personally not have it anywhere, given that there's always `MetadataMediator::getSceneAttributesByName :` and the like, to me it looks like the message alone tells the location quite well on its own :)

Reply:
Ok.  I was thinking about getting rid of that message prefix (I manually add that everywhere heh), but the bang output is a bit long, so i'll get rid of the bang :)

Reply:
Ooof.  This log messaging refactor really shouldn't be a part of this PR anyway.  I accidentally merged my shader type branch onto my message refactor branch instead of master.  I am going to remake this PR properly. Sorry!